platform/android/java/editor/src/main/java/org/godotengine/editor/BaseGodotGame.kt

```diff
@@ -103,7 +103,7 @@ abstract class BaseGodotGame: GodotEditor() {
 
 	@CallSuper
 	override fun supportsFeature(featureTag: String): Boolean {
-		if (HYBRID_APP_FEATURE == featureTag) {
+		if (HYBRID_APP_FEATURE == featureTag && godot?.isXrRuntime == true) {
 			// Check if hybrid is enabled
 			return isHybridAppEnabled()
```

Comment:
Super nit-picky, but I feel like this should be:

```c++
		if (HYBRID_APP_FEATURE == featureTag) {
			// Check if hybrid is enabled
			return godot?.isXrRuntime == true && isHybridAppEnabled()
		}
```

The `if` is just establishing which feature tag, but the `godot?.isXrRuntime == true` is actually part of the check if that feature tag is enabled